javascript/features/communication_commands/communication_commands.js

```diff
@@ -11,7 +11,7 @@ import { MuteCommands } from 'features/communication_commands/mute_commands.js'
 
 // Set of `/show` messages that Gunther will issue at a particular interval.
 const kGuntherMessages = [
-    'beg', 'discord', 'donate', 'forum', 'irc', 'minigames', 'reg', 'report', 'rules', 'stunt',
+    'beg', 'discord', 'dm', 'donate', 'forum', 'irc', 'minigames', 'reg', 'report', 'rules', 'stunt',
```

Comment:
Lines cannot be longer than a hundred characters. Please wrap this, like `weapons` already made it to the next line.

Reply:
Sorry, Forgot about it. Done it though!